src/utils/IgnoredFiles.ts

```diff
@@ -2,19 +2,21 @@ import * as yaml from 'js-yaml';
 import * as fs from 'fs';
 // import { minimatch } from 'minimatch';
 import * as minimatch from 'minimatch';
+import * as path from 'path';
+import { actionOptions } from './ActionOptions';
 
 /**
  * The ignore files in the analysis_options.yaml
  */
-export class IgnoredFiles{
+export class IgnoredFiles {
   private readonly patterns: minimatch.IMinimatch[];
   constructor() {
     let patterns: string[];
     try {
-      const yamlFile = yaml.load(fs.readFileSync('./analysis_options.yaml', 'utf8')) as {analyzer?: {exclude?: string[]}};
+      const yamlFile = yaml.load(fs.readFileSync(path.resolve(actionOptions.workingDirectory, './analysis_options.yaml'), 'utf8')) as { analyzer?: { exclude?: string[] } };
```

Comment:
Shouldn't it be

```suggestion
      const yamlFile = yaml.load(fs.readFileSync(path.resolve(actionOptions.workingDirectory, 'analysis_options.yaml'), 'utf8')) as { analyzer?: { exclude?: string[] } };
```

instead?